libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -897,6 +988,47 @@ def Delete(self, delete_disks: bool = False) -> None:
             self.FormatLogMessage(
                 'Could not find disk: {0:s}, skipping'.format(disk_name)))
 
+  def SetTags(self, new_tags: List[str]) -> None:
+    """Sets network tags for the compute instance.
+
+    Args:
+        new_tags (List[str]): A list of tags. Each tag must be 1-63
+            characters long, and comply with RFC1035.
+
+    Raises:
+      InvalidNameError: If the name of the snapshot does not
```

Comment:
Do you mean `tags` instead of `snapshot`?

Reply:
yes it is tags, I was so lazy and copied part of the error description from another docstring